index.html

```diff
@@ -388,9 +388,101 @@ <h2 id="protocol-registry">
+      [=dictionary=] to which the {{DigitalCredentialsProvider}}'s
+      {{DigitalCredentialsProvider/request}} is [=converted to idl
+      values|converted=] to before it is passed onto underlying platform).
+      </li>
+      <li>MUST define a [[WebIDL]] [=dictionary=] representation of the
+      [=digital credential/exchange protocol=] response structure (i.e., the
+      [=dictionary=] to which the {{DigitalCredential}}'s
+      {{DigitalCredential/data}} is [=converted to idl values|converted=] before it
+      is made available to the relying party).
+      </li>
+      <li>MUST define validation rules for members of the request and response
+      structures.
+      </li>
+      <li>MUST have undergone privacy review by the W3C's Privacy Interest
+      Group and Federated Identity Working Group.
+      </li>
+      <li>MUST have undergone security review by the Federated Identity Working
+      Group.
+      </li>
+      <li>MUST have implementation commitment from at least two implementers in
+      independent browser engines, to meet the W3C's adequate implementation
+      experience requirements.
+      </li>
+      <li>MUST have formally recorded consensus by the Working Group to be
+      included in the registry.
+      </li>
+    </ol>
+    <p>
+      Once the above criteria are met, the protocol will be included in the
+      registry.
+    </p>
+    <h3>
+      Change process
+    </h3>
     <p>
-      To be included in the registry...
+      To add a new [=digital credential/exchange protocol=] to the registry, or
+      to update an existing one:
     </p>
+    <dl>
+      <dt>
+        Define a protocol identifier
+      </dt>
+      <dd>
+        The protocol identifier MUST be a unique string that is not already in
+        use in the registry. Use only lowercase ASCII letters, digits, and
+        hyphens (e.g., "protocol", "the-protocol"). Avoid using version numbers
+        in the protocol identifier.
+      </dd>
+      <dt>
+        Link to a Web IDL request dictionary
+      </dt>
+      <dd>
+        The <dfn>Web IDL request dictionary</dfn> MUST be a [=dictionary=] that
+        defines the structure of the request that is passed, via
+        {{DigitalCredentialsProvider}}'s
+        {{DigitalCredentialsProvider/request}}, to the holder's a digital
+        wallet.
+      </dd>
+      <dt>
+        Link to a Web IDL response dictionary
+      </dt>
+      <dd>
+        The <dfn>Web IDL response dictionary</dfn> MUST be a [=dictionary=]
+        that defines the structure of {{DigitalCredential}}'s
+        {{DigitalCredential/data}}.
+      </dd>
+      <dt>
+        Describe the protocol
+      </dt>
+      <dd>
+        The description MUST be a brief summary of the protocol's purpose and
+        use case.
+      </dd>
+      <dt>
+        Provide a link to the specification
+      </dt>
+      <dd>
+        The specification MUST be a stable URL that points to the authoritative
```

Comment:
I'd like to also include some criteria about the authoritative source at that URL being open and freely available.

There's been a lot of talk about developer ergonomics in the context of these protocol identifiers. One of the least ergonomic things a developer can run into is hitting a pay-walled document when just trying to do their job.

Reply:
@bc-pi, how does this sound? https://github.com/WICG/digital-credentials/pull/157/commits/296438428e10288d029115be9c921e36d4086537

Reply:
We can't force "freely" available, unfortunately. ISO probably won't do it. At least it is publicly available.

Reply:
> We can't force "freely" available, unfortunately.

Why not?

> ISO probably won't do it. 

ISO or any other SDO could absolutely chose to make a specification available freely and publicly at a URL.

Reply:
> @bc-pi, how does this sound? [2964384](https://github.com/WICG/digital-credentials/commit/296438428e10288d029115be9c921e36d4086537)

Apologies for the slow response, I was on vacation with my family. But thank you, that text sounds good.

Reply:
> We can't force "freely" available, unfortunately. ISO probably won't do it. At least it is publicly available.

Isn't this a chicken and egg problem - if the requirement for inclusion in the registry is freely available, then ISO probably would do it?

Reply:
Depends how much they want to be part of the new hotness that is the DC API.